RELEASE.md

```diff
@@ -1,34 +1,60 @@
 # Release Process
 
-## Requirements
+This documentation guides you on how to release a new version of the Codacy Chart.
 
-Please make sure you have these tools installed before starting this process:
+The release process consists of the following stages:
 
--   git
--   [ytool](https://github.com/codacy/ytool)
--   [helm >=3.x](https://helm.sh/docs/intro/install/)
+1.  **Deciding to release a new version**
 
-## Prepare a new release
+    Decide that we're ready to release a new version of the Codacy Chart and elect who will be the Release Manager.
 
--   [ ] 1.  Inform the engineering team that you are the release manager for a new release in #engineering in our slack (tag engineers) and get a status from each squad on any changes that may be on the master branch of components that may be dealbreakers for the release.
+2.  **Preparing the release for testing**
 
--   [ ] 2.  Clone this project on master branch
+    The Release Manager creates a release candidate branch that will be used to test the new release.
+
+3.  **Testing and stabilizing the release**
+
+    The Release Manager involves the QA team and relevant stakeholders in testing the new release, and applies fixes for bugs that are detected.
+
+4.  **Launching the new release**
+
+    The Release Manager releases and announces the new version.
+
+The next sections include detailed instructions on how to complete each step of the release process. Make sure that you complete each step before advancing to the next one.
 
--   [ ] 3.  Decide the new version
+## 1. Deciding to release a new version
```

Comment:
In my opinion, this first section is missing important details. For example

* What is the process for deciding when to release a new version?
* How do we decide if the new version is a major, minor, or patch version?
* How is the Release Manager "elected"?

Reply:
> How do we decide if the new version is a major, minor, or patch  version?

I like this description from semver: https://semver.org/
```
Given a version number MAJOR.MINOR.PATCH, increment the:

    MAJOR version when you make incompatible API changes,
    MINOR version when you add functionality in a backwards compatible manner, and
    PATCH version when you make backwards compatible bug fixes.
```

What do you think about putting something like this here?

Like:
```
Given a version number MAJOR.MINOR.PATCH, increment the:

    MAJOR version when you make changes that are incompatible with old values.yaml files or integrations.
    MINOR version when you add functionality in a backwards compatible manner, and
    PATCH version when you make backwards compatible bug fixes.
```

Reply:
I agree!

Reply:
Already added this information. :+1: